src/github/github-api.spec.js

```diff
@@ -4,8 +4,10 @@ import GitHubApi from "./github-api";
 import {githubClientId as clientId,
     githubClientSecret,
     githubAccessToken as accessToken,
-   githubClientUsername as username,
-   testRepoName} from "../../test.config.js";
+        testRepoName,
+        testGithubClientUsername as testUsername,
+        testPublickRepo,
+        testPrivateRepo} from "../../test.config.js";
```

Comment:
Need to add this to text.config.example as well: ) use these values in that example since they are not sensitive info.